backend/src/main/java/feedzupzup/backend/feedback/application/AdminFeedbackService.java

```diff
@@ -68,4 +70,21 @@ public AdminFeedbackListResponse getFeedbackPage(
         feedbackLikeCounter.applyBufferedLikeCount(feedbackPage);
         return AdminFeedbackListResponse.from(feedbackPage);
     }
+
+    @Transactional
+    @BusinessActionLog
+    public UpdateFeedbackCommentResponse updateFeedbackComment(
+            final UpdateFeedbackCommentRequest request,
+            final Long feedbackId
+    ) {
+        final Feedback feedback = getFeedback(feedbackId);
+        feedback.updateComment(request.comment());
+        feedback.updateStatus(CONFIRMED);
```

Comment:
p1)
댓글을 남기면 완료상태로 변경되어야 한다는 저희의 도메인 로직을 지키기 위해 하나의 메서드로 묶으면 좋을 것 같아요!
원자성 지키기 위해!!
```suggestion
        feedback.writeComment(request.comment());
```